python_ta/cfg/graph.py

```diff
@@ -5,12 +5,12 @@
 try:
     from z3 import Z3_OP_UNINTERPRETED, ExprRef, Not, Z3Exception, is_const
 
-    from ..transforms.ExprWrapper import ExprWrapper, Z3ParseException
+    from python_ta.z3.z3_parser import Z3ParseException, Z3Parser
```

Comment:
You should still use a relative import here (`..z3.z3_parser`) because it's within the same package.

This is true for all of the imports that you've added throughout